.gitignore

```diff
@@ -39,3 +39,6 @@ csvs/
 load-dotenv.sh
 push_csvs.py
 .db_snapshots/
+
+# Data dump for docker postgres
+docker_postgres_dump.sql
```

Comment:
How was this file generated? Classic data dump of all postgres tables?

How do we keep this updated? Is this something where a dump will have to be manually generated every time someone new wants to get Docker set up?